plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -55,10 +86,15 @@ const getProductData = () => {
 		 )?.value;
 	}
 
+	const productTypeOptions = getProductTypeOptions();
```

Comment:
Nitpick: since in all occurrences you're having to call these two functions to get what you want, maybe you could have a function that combines the two, or even combine the two functions into one.

Reply:
I debated this but ended up going with this implementation to keep the functions as small and focused as possible. I think it makes ready and understanding the code a bit clearer and also makes debugging a bit easier.

I'll keep this as is for now. It is something that could always be re-considered when in this part of the code in the future.